### PR TITLE
Add job for closing stale issues and PRs

### PR DESCRIPTION
**Pull request by violetagg**

### CODE REVIEW

File: .github/workflows/close-stale-issues-prs.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
```
Comment (violetagg) [inline .github/workflows/close-stale-issues-prs.yml:17]:
```suggestion
    - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e
```

File: .github/workflows/close-stale-issues-prs.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e
+      with:
+        stale-issue-message: 'If you would like us to look at this issue, please provide the requested information. If the information is not provided within the next 7 days this issue will be closed.'
+        stale-pr-message: 'If you would like us to look at this PR, please provide the requested information. If the information is not provided within the next 7 days this PR will be closed.'
+        close-issue-message: 'Closing due to lack of requested feedback. If you would like us to look at this issue, please provide the requested information and we will re-open.'
+        close-pr-message: 'Closing due to lack of requested feedback. If you would like us to look at this, please provide the requested information and we will re-open.'
```
Comment (pderop) [inline .github/workflows/close-stale-issues-prs.yml:22]:
```suggestion
        close-pr-message: 'Closing due to lack of requested feedback. If you would like us to look at this PR, please provide the requested information and we will re-open.'
```

**Review by pderop — APPROVED** [review]
apart from the small suggested change about the close-pr-message, +1

(useful PR !)

**Comment by violetagg** [comment]
@pderop Thanks for the review!

**Pull request merged**